account_asset_management_method_number_end/tests/test_account_asset_management.py

```diff
@@ -0,0 +1,77 @@
+# Copyright 2009-2018 Noviat.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+import time
+
+import odoo.tests.common as common
+from odoo import tools
+from odoo.modules.module import get_resource_path
+
+
+class TestAssetManagement(common.TransactionCase):
+
+    def _load(self, module, *args):
+        tools.convert_file(self.cr, module,
+                           get_resource_path(module, *args),
+                           {}, 'init', False, 'test',
+                           self.registry._assertion_report)
+
+    def setUp(self):
```

Comment:
do you really need to load this for every test? Otherwise pls use SavepointCase + setUpClass